### DISCUSSION TITLE
How can I improve XGBoost percentage accuracy?

### DISCUSSION THREAD

**Discussion by sonymag** [kyleskom/NBA-Machine-Learning-Sports-Betting#233] [Q&A] [answered]
@kyleskom Hello man! At first, thx u for this project, u a the best!
So, in project folder `Models/XGBoost Models` I found **XGBoost_54.8%_UO-8.json** and **XGBoost_68.6%_ML-2.json**
Can the rate be improved? Maybe it possible to get **74.6%** for _OU_ and _ML_.
Any help or advice is appreciated. By the way, tell me if u need get more odds, I`m scraper monster)
With many thanks!

**Comment by onlyadamscott** [comment, marked as answer]
I have models trained at 72% but you must consider a point of diminishing returns. Make sure you test for accuracy.

**Reply by sonymag** [reply]
Ok, I will sir, thx !!!

**Reply by onlyadamscott** [reply]
Meaning, even with highky trained models, there is a sweet spot.

**Reply by onlyadamscott** [reply]
Depends which model. I'm running XGBoost. In theory this is a good model. But I can push to 72% doesnt matter. My model at 70% had the same efficiency

**Reply by sonymag** [reply]
interesting..

**Reply by rohitvishnubhatla** [reply]
how do you test for efficiency?

**Reply by roryosullivan03** [reply]
@onlyadamscott How did you increase the percent?

**Comment by JP23000** [comment]
But why do you need a higher %?

That 54% XGBoost hit for me like 62% of the time over the last 3 months of the season.

But lets say it stays at 54%.  You only need 52.5% to be profitable.  So at the 54 mark you are still Vegas Profitable + 2.8%

Running that through 100 games would have you winning 55 games and losing 45 games.  If you bet $100 a game and win 55 games at a -110 odds you will get 5k in winnings.  Minus the 4500 in losses from the other games nets you out at a $500 gain per 100 games bet on.  Or 5 units.  There are 1230 games per year.  So multiply that but 12.3 and you get $6150 in profits per season per $100 bet.

If you utilize that 1 unit = 1% method of bankroll management you are looking at a 61.5% uncompounded growth each season.

I work in the stock market and we have a saying.... sometimes the bears win and sometimes the bulls win.. but the pigs always get slaughtered.

Don't get greedy... it backfires most times.

**Reply by sonymag** [reply]
So, the main  its - if u will win 55 games at a -110 odds))

**Reply by rpassaro** [reply]
So if you just keep training the model, the accuracy will improve? Like just running the training commands?
